appservice/src/intTest/groovy/nu/yona/server/PinResetRequestTest.groovy

```diff
@@ -72,14 +81,33 @@ class PinResetRequestTest extends AbstractAppServiceIntegrationTest
 		appService.deleteUser(richard)
 	}
 
-	def 'Clear pin reset request before verification'()
+	def 'Try to verify pin reset confirmation code before end of delay period'()
 	{
 		given:
 		User richard = addRichard()
 		appService.yonaServer.postJson(richard.pinResetRequestUrl, [:], ["Yona-Password" : richard.password])
 		richard = appService.getUser(appService.&assertUserGetResponseDetailsPublicDataAndVpnProfile, richard.url, true, richard.password)
```

Comment:
Is this line needed? Only the URL/password are used and those are the same isn't it?